routes/provide.go

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2020 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package routes
+
+import (
+	"net/http"
+
+	"github.com/xmidt-org/mimisbrunnr/eventParser"
+	"github.com/xmidt-org/mimisbrunnr/manager"
+	"github.com/xmidt-org/mimisbrunnr/model"
+	"github.com/xmidt-org/mimisbrunnr/norn"
+	"go.uber.org/fx"
+)
+
+type HandlerIn struct {
+	fx.In
+
+	Registry *norn.Registry
+
+	Manager *manager.Manager
+
+	EventParser *eventParser.EventParser
```

Comment:
We should expect interfaces as dependencies instead of pointers to structs (it makes the different structs more loosely coupled and have a defined api).  It's okay (and probably for the best) to declare those interfaces here, just including the functions required.

Reply:
Now that I've looked at this more, it looks like just a translator to create the functions expected elsewhere.  I need @kcajmagic to look at this and give some guidance.  It feels like there should be a cleaner way to do this (ie just expect the interfaces I asked for in routes.go)